src/firecracker/src/main.rs

```diff
@@ -94,105 +78,108 @@ fn main() {
         }
     }));
 
-    let cmd_arguments = App::new("firecracker")
-        .version(crate_version!())
-        .author(crate_authors!())
-        .about("Launch a microvm.")
-        .arg(
-            Arg::with_name("api_sock")
-                .long("api-sock")
-                .help("Path to unix domain socket used by the API")
-                .takes_value(true)
-                .default_value(DEFAULT_API_SOCK_PATH),
-        )
+    let mut app = App::new().name("Firecracker").version(FIRECRACKER_VERSION)
+        .header("Launch a microVM.")
         .arg(
-            Arg::with_name("id")
-                .long("id")
-                .help("MicroVM unique identifier")
-                .takes_value(true)
-                .default_value(DEFAULT_INSTANCE_ID)
-                .validator(|s: String| -> Result<(), String> {
-                    validate_instance_id(&s).map_err(|e| format!("{}", e))
-                }),
-        )
-        .arg(
-            Arg::with_name("seccomp-level")
-                .long("seccomp-level")
-                .help(
-                    "Level of seccomp filtering.\n
-                            - Level 0: No filtering.\n
-                            - Level 1: Seccomp filtering by syscall number.\n
-                            - Level 2: Seccomp filtering by syscall number and argument values.\n
-                        ",
-                )
+            ArgInfo::new("api-sock")
                 .takes_value(true)
-                .default_value("2")
-                .possible_values(&["0", "1", "2"]),
-        )
-        .arg(
-            Arg::with_name("start-time-us")
-                .long("start-time-us")
-                .takes_value(true)
-                .hidden(true),
-        )
-        .arg(
-            Arg::with_name("start-time-cpu-us")
-                .long("start-time-cpu-us")
-                .takes_value(true)
-                .hidden(true),
-        )
-        .arg(
-            Arg::with_name("config-file")
-                .long("config-file")
-                .help("Path to a file that contains the microVM configuration in JSON format.")
-                .takes_value(true)
-                .required(false),
-        )
-        .arg(
-            Arg::with_name("no-api")
-                .long("no-api")
-                .help("Optional parameter which allows starting and using a microVM without an active API socket.")
-                .takes_value(false)
-                .required(false)
-                .requires("config-file")
-        )
-        .get_matches();
-
-    let bind_path = cmd_arguments
-        .value_of("api_sock")
+                .default_value(DEFAULT_API_SOCK_PATH)
+                .help("Path to unix domain socket used by the API"),
+        ).arg(
+        ArgInfo::new("id")
+            .takes_value(true)
+            .default_value(DEFAULT_INSTANCE_ID)
+            .help("MicroVM unique identifier"),
+    ).arg(
```

Comment:
The formatting here looks wrong and `cargo fmt` doesn't seem to catch this error. Can you manually format it please?